### DISCUSSION TITLE
Provide support for RecordNameStrategy

### DISCUSSION THREAD

**Discussion by MrdjanPoletanovic** [provectus/kafka-ui#3353] [Ideas]
Hi,
Currently, in the topic messages section, it is not possible to decode key and value if the naming strategy is RecordNameStrategy (the fully-qualified record name regardless of the topic).
Is there anything I can try to enable this feature?

**Comment by HyundeokPark** [comment]
perhaps, you could find something useful [here](https://github.com/provectus/kafka-ui/discussions/3349#discussion-4861304)

**Reply by MrdjanPoletanovic** [reply]
This type of strategy is not same as TopicRecordNameStrategy. I am using master image and messages cannot be decoded.

**Comment by Haarolean** [comment]
We're gonna work on this issue pretty soon

**Comment by Haarolean** [comment]
#1361
